backend/run_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Run phonebook backend tests.
+
+cd $(dirname $0)/import
+
+venv/bin/pytest --flake8
```

Comment:
You'll want to add a Jenkinsfile for running the tests, and make it generate XML output to feed back to Jenkins. Also, coverage data is nice!

Reply:
Good idea!

Reply:
I made a pretty bare-bones one for `flask_saml_sso` that might be useful for this project as well.
https://github.com/magenta-aps/flask_saml_sso/blob/development/Jenkinsfile